<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertAsWindow</class>
 <widget class="QMainWindow" name="InsertAsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insertar Aseguradora</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 255, 206);</string>
    </property>
    <property name="text">
     <string>Telefono:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_insertar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        background-color: #FFC107;
        color: black;
        border: none;
        border-radius: 5px;
        padding: 10px;
      }</string>
    </property>
    <property name="text">
     <string>Insertar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 255, 235);</string>
    </property>
    <property name="text">
     <string>Nombre de la Aseguradora:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_nombre">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
          border: 2px solid #ffffff;
          border-radius: 10px;
          padding: 8px;
          font-size: 14px;
          background-color: rgb(255, 249, 213);
      }</string>
    </property>
    <property name="placeholderText">
     <string>Nombre de Aseguradora</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_telefono">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
          border: 2px solid #ffffff;
          border-radius: 10px;
          padding: 8px;
          font-size: 14px;
		  background-color: rgb(255, 249, 213);
      }</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Telefono</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 255, 206);</string>
    </property>
    <property name="text">
     <string>Direccion</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_direccion">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
          border: 2px solid #ffffff;
          border-radius: 10px;
          padding: 8px;
          font-size: 14px;
		  background-color: rgb(255, 249, 213);
      }</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Direccion</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>506</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
